monitoring/README.md

```diff
@@ -92,47 +175,124 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller*, *status-addon-controller* and *status-agent-controller*. 
 
-- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+- `Instance`: hostname and port number for the target KubeStellar space controller, e.g., *10.244.0.71:8090*
 
     <img src="images/ks-controllers-monitoring.png" width="60%" height="80%" title="ks-controllers-monitoring">
 
-To monitor the KubeStellar controllers in a WDS space (e.g., wds1) use the following parameters: 
-- `SpaceNS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
-- `SpaceNS: wds1-system` and `KSController: ks-transport-controller`
+To monitor the KubeStellar controllers in a WDS space (e.g., wds1) or ITS space (e.g., its1) use the following combination of `NS/APIService` parameters: 
+- `NS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
+- `NS: wds1-system` and `KSController: ks-transport-controller`
+
+Likewise, to monitor the KubeStellar controller in a ITS space and WEC (e.g., its1): `NS: its1-system` and `KSController: status-addon-controller` & `NS: open-cluster-management-agent-addon` and `KSController: status-agent-controller`.
+
+Note: 
+- Use the following commands to obtain the Ip address of the KubeStellar controllers in the WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces and WEC (e.g.,`cluster1`):
+
+  ```bash
+  kubectl --context kind-kubeflex -n wds1-system get Endpoints | egrep ".*controller"
+  ```
+  ```bash
+  kubectl --context kind-kubeflex -n its1-system get Endpoints | egrep ".*controller"
+  ```
+  ```bash
+  kubectl --context cluster1 -n open-cluster-management-agent-addon get Endpoints
+  ```
+
+Here are the default port number for the KubeStellar Controllers metrics endpoint: 
+
+| KS Controllers            | Default Port #|
+| ------------------------- | ------------- |
+| KS controller-manager     |     8443      |
+| transport-controller      |     8090      |
+| status-addon-controller   |     9280      |
+| status-agent-controller   |     8080      |
 
-Likewise, to monitor the KubeStellar controller in a ITS space (e.g., its1): `SpaceNS: its1-system` and `KSController: status-addon-controller`
 
 
 #### 6. View Pyroscope profile graphs for KubeStellar controllers in Grafana: 
 
 a) Connect to Pyroscope dashboard: 
-    i) Click `Connections -> Data Sources` in the primary menu
-    ii) Click `Explore` in the Pyroscope datasource selection
+   i) Click `Connections -> Data Sources` in the primary menu
+  ii) Click `Explore` in the Pyroscope datasource selection
 
```

Comment:
FYI, I get a lot of failures when I try the Pyroscope stuff. They seem to be transient; I see no container restarts or Pod replacements. But here is what I see in my shell running the main port-forward. The errors come only from the Pyroscope part.

```console
mspreitz@mjs13 monitoring % kubectl --context kind-kubeflex port-forward -n ks-monitoring svc/grafana 3000:80
Forwarding from 127.0.0.1:3000 -> 3000
Forwarding from [::1]:3000 -> 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
E0929 21:22:00.412418   51571 portforward.go:381] error copying from remote stream to local connection: readfrom tcp4 127.0.0.1:3000->127.0.0.1:52873: write tcp4 127.0.0.1:3000->127.0.0.1:52873: write: connection reset by peer
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
E0929 21:22:36.976687   51571 portforward.go:381] error copying from remote stream to local connection: readfrom tcp4 127.0.0.1:3000->127.0.0.1:52890: write tcp4 127.0.0.1:3000->127.0.0.1:52890: write: broken pipe
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
Handling connection for 3000
E0929 21:23:06.621547   51571 portforward.go:370] error creating forwarding stream for port 3000 -> 3000: Timeout occurred
E0929 21:23:06.972897   51571 portforward.go:347] error creating error stream for port 3000 -> 3000: Timeout occurred
E0929 21:23:07.088161   51571 portforward.go:370] error creating forwarding stream for port 3000 -> 3000: Timeout occurred
E0929 21:23:07.090158   51571 portforward.go:347] error creating error stream for port 3000 -> 3000: Timeout occurred
Handling connection for 3000
E0929 21:23:28.066825   51571 portforward.go:370] error creating forwarding stream for port 3000 -> 3000: Timeout occurred
Handling connection for 3000
E0929 21:23:49.321673   51571 portforward.go:347] error creating error stream for port 3000 -> 3000: Timeout occurred
Handling connection for 3000
E0929 21:25:00.695454   51571 portforward.go:370] error creating forwarding stream for port 3000 -> 3000: Timeout occurred
E0929 21:25:42.538100   51571 portforward.go:381] error copying from remote stream to local connection: readfrom tcp4 127.0.0.1:3000->127.0.0.1:52896: write tcp4 127.0.0.1:3000->127.0.0.1:52896: write: broken pipe
Handling connection for 3000
Handling connection for 3000
E0929 21:26:19.237958   51571 portforward.go:347] error creating error stream for port 3000 -> 3000: Timeout occurred
```